cli/command/registry/login.go

```diff
@@ -47,6 +51,27 @@ func runLogin(dockerCli command.Cli, opts loginOptions) error {
 	ctx := context.Background()
 	clnt := dockerCli.Client()
 
+	if opts.password != "" {
+		fmt.Fprintln(dockerCli.Err(), "WARNING! Using --password via the CLI is insecure. Use --password-stdin")
+		if opts.passwordStdin {
+			return errors.Errorf("--password and --password-stdin are mutually exclusive")
```

Comment:
`errors.New`

And I'm wondering if there's a way to phrase this that's more friendly to people who don't speak english as a first language, but I don't have any good ideas right now.